### ISSUE TITLE
Can't compile "World" script module - Multiple declaration of function 'CTRL'

### ISSUE THREAD

**Issue by matellis** [SchnitzelPommes/DayZ-SA-Tomato#30]
Dayz is throwing an error on startup with this mod. The dialog says:
```
Can't compile 'World' script module!
   
JM/COT/Scripts/4_World/staticfunctions.c(366): Multiple declaration of function 'CTRL'`
```
In `addons/DayZ-SA-Tomato/scripts/4_World/COMCam.c` the `CTRL` function is repeated, with a comment *"static functions arent scope global?"*

Even when I comment this function out I get the error. A grep shows the PBO file contains the word "CTRL" so (knowing very little about PBO files) I suspect the duplicate function is also in there and my edits do very little. 

tl;dr> Looks like static functions are now scope global and one version of `CTRL` has to go!
